src/data/items.h

```diff
@@ -1445,7 +1445,7 @@ const struct Item gItems[] =
         .description = sDynamaxCandyDesc,
         .pocket = POCKET_ITEMS,
         .type = ITEM_USE_BAG_MENU,
-        .fieldUseFunc = ItemUseOutOfBattle_CannotUse, // Todo
+        .fieldUseFunc = ItemUseOutOfBattle_CannotUse,
```

Comment:
Restore this "// Todo". This PR doesn't handle the Dynamax Candy's effect, and it's still not implemented as a result of not having Dynamax mechanics implemented.